echo-model/src/main/java/com/netflix/spinnaker/echo/model/Pipeline.java

```diff
@@ -100,5 +100,13 @@
 
   @JsonPOJOBuilder(withPrefix = "")
   public static final class PipelineBuilder {
+    @JsonProperty("config")
+    private void unpackConfig(Map<String, Object> config) {
```

Comment:
how does this get called? is this `jackson` "magic"?

Reply:
maybe more appropriate/idiomatic to call this method `setConfig`?

Reply:
ah, this is symmetric to the one in `orca`, got it. still slightly prefer the `set` idiom

Reply:
I'm going to keep it the same as we have in orca. Also, "unpack" to me implies that it's doing more than setting the field, it's also doing other things.

Reply:
And yes, when the object is deserialized this gets magically called.